content/v2.0/reference/glossary.md

```diff
@@ -439,11 +439,11 @@ Related entries: [explict block](#explicit-block), [block](#block)
 
 ### influx
 
-A command line interface (CLI) that interacts with the InfluxDB daemon (influxd).
+`influx` is a command line interface (CLI) that interacts with the InfluxDB daemon (influxd).
```

Comment:
This is a little different than the style elsewhere in the glossary, but I think the visual cue (`influx` and `influxd`) is helpful to the reader here.